### ISSUE TITLE
Feature Request: wildcard for include files

### ISSUE THREAD

**Issue by vanrein** [bstansell/conserver#47]
Most Linux daemons have adopted an include-file approach like `/etc/conserver.d/*.cf` which can load files installed by various packages that are independently installed.  In a containerised use case, the equivalent could be container descriptions that are independently configured with a TTY.

*Apologies for the small burst of Feature Requests.  They are of course related, but may be easier to you to consider separately.  I looked into offering pull requests but got stuck in autotools, for which I found no explanation or bootstrapping script.*

**Comment by bstansell** [comment]
Yeah, increasing the `#include` directive to support wildcards would certainly be a nice addition.  That would allow for a master config file to just have `#include /etc/conserver.d/*.cf` and pull in anything generated.

As for the autotools bits, yeah, it's basically undocumented.  The `.cirrus.yml` file gives hints (as it needs a working environment to build things) and it basically just makes sure the `autoconf` and `automake` packages are installed (for freebsd, the linux image has them already) and then runs `./package/setup-configure`.
